tests/test_distributions.py

```diff
@@ -388,6 +594,24 @@ def test_int_log_uniform_distribution_asdict() -> None:
     assert EXAMPLE_DISTRIBUTIONS["ilu"]._asdict() == {"low": 2, "high": 12, "step": 2}
 
 
+def test_int_invalid_step() -> None:
+
+    with pytest.raises(ValueError):
+        distributions.IntDistribution(low=1, high=100, step=0)
+
+    with pytest.raises(ValueError):
+        distributions.IntDistribution(low=1, high=10, step=-1)
+
+
+def test_float_invalid_step() -> None:
+
+    with pytest.raises(ValueError):
+        distributions.FloatDistribution(low=1.0, high=10.0, step=0)
+
+    with pytest.raises(ValueError):
+        distributions.FloatDistribution(low=1.0, high=100.0, step=-1)
+
+
```

Comment:
Could add unittests for ValueError for `FloatDistribution` and `IntDistribution`?